### PR TITLE
i moved the alert box to the top of the homepage

### LINKED ISSUE #8989: [BUG] <Move Alert Box to the Top of the Page in Dev Mode>
### Description

In dev mode, when there's no data loaded, we currently have an alert box on the home page. However, it's not appearing at the  location, which is at the top of the page under the main navbar. This issue is about relocating the alert box to the top.

### Screenshots

alert

### Do you want to work on this issue?

Yes

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Nwakaego-Ego**
## Fixes Issue

Edit: 
Fixes #8989 

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Review by sital002 — COMMENTED** [review]
The package-lock file shouldn't be changed unless you install some dependencies so please remove the package-lock file and package.json.

File: components/Alert.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ export default function Alert({ type, message }) {
     case "warning":
       return (
         <div
-          className={`alert-${type} mt-3 bg-yellow-100 rounded-lg py-5 px-6 mb-3 text-base text-yellow-700 inline-flex items-center w-full`}
+          className={`alert-${type}  bg-yellow-100 rounded-lg py-5 px-6 mb-3   text-base text-yellow-700 inline-flex items-center w-full`}
```
Comment (sital002) [inline components/Alert.js:30]: If we remove the margin top, the component will snap with the element above so instead use my property

File: pages/index.js [hunk 2]
```diff
@@ -239,39 +267,10 @@ export default function Home({
         </div>
       </div>
 
-      <div className="bg-primary-low dark:drop-shadow-none dark:bg-dark mb-8 p-8 drop-shadow-md">
-        {alerts.map((alert, index) => (
-          <Alert key={index} type={alert.type} message={alert.message} />
-        ))}
```
Comment (eddiejaoude) [inline pages/index.js:245]: Thank you for your contribution 👍  

But only this alert was meant to be moved, not the statistics...

Screenshot 2023-09-09 at 04 23 07

File: package.json [hunk 3]
```diff
@@ -116,5 +116,9 @@
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/EddieHubCommunity/BioDrop/issues"
+  },
+  "main": "playwright.config.js",
+  "directories": {
+    "test": "tests"
```
Comment (eddiejaoude) [inline package.json:122]: Are these changes needed?

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Thank you for the contribution 👍 

I have left some inline comments

File: components/Alert.js (hunk 1 above)
Comment (Nwakaego-Ego) [reply components/Alert.js:30]: After cloning the project, when attempting to run it using npm run dev, I received the following error message: 

image

As a result of this issue, I proceeded to install the Node.js package by running the "npm install"
I would appreciate it if you could provide guidance on how to run it in my terminal without the need to download any additional dependencies.

"If we remove the margin top, the component will snap with the element above so instead use my property" I have made this adjustment.

Comment (sital002) [reply components/Alert.js:30]: run ```npm ci ``` instead of ```npm install```. Eddie had made a video on it you can check that out for reference.
https://www.youtube.com/watch?v=jHkIXYVpMTo&ab_channel=EddieJaoude .You can ask question on our discord server for any kind of help.

Comment (Nwakaego-Ego) [reply components/Alert.js]: Thank you for the video, I was able to figure it out

**Comment by eddiejaoude** [comment]
Sorry @Nwakaego-Ego I really needed the changes to show an alert to our users, so made the change. I will close this pull request and issue, please can you select another issue to work on. Thanks
